bioio/writers/__init__.py

```diff
@@ -2,9 +2,11 @@
 # -*- coding: utf-8 -*-
 
 from .ome_tiff_writer import OmeTiffWriter
-from .ome_zarr_writer import OmeZarrWriter
+from .ome_zarr_writer import OmeZarrWriter as OmeZarrWriter_old
+from .ome_zarr_writer_2 import OmeZarrWriter
 
 __all__ = [
     "OmeTiffWriter",
     "OmeZarrWriter",
+    "OmeZarrWriter_old",
 ]
```

Comment:
Is our old reader better for large Z stack data while this one is better for timeseries or is this one just better overall?

Reply:
Good question.  (Note this is still a draft PR and I am not expecting that this `_old` suffix is really best.  It's less of a breakage to call the new one OmeZarrWriter2 and leave the old one with same name.)

Here are some of the key differences:
* The new one is built on tooling for computing multiscales that is more stable than what the old one uses
* The new one works better for bigger data because it does things in smaller steps. There is no one one-size-fits-all "write this data array to this url" function in the new writer -- at least not yet.   
* The new code is coming from another repo where it's been used to do some bulk data conversions internally already.  
* The new one is currently providing functions that expect to deal with time series of zstacks where the zstacks still might reasonably fit in memory.  
* The new one is currently assuming 5D data coming from bioio.BioImage, or sequences of images (one per T), or (in progress) a 5D ArrayLike.